packages/trader/src/Modules/Trading/Containers/chart-widgets.tsx

```diff
@@ -48,16 +49,13 @@ export const DigitsWidget = observer(({ digits, tick }: { digits: TDigits['digit
 });
 
 // Chart widgets passed into SmartCharts
-export const ChartTopWidgets = observer(({ charts_ref, open_market, open }: TChartTopWidgets) => {
+export const ChartTopWidgets = observer(({ open_market, open }: TChartTopWidgets) => {
     const { client, ui } = useStore();
     const { is_digits_widget_active, onChange: onSymbolChange } = useTraderStore();
     const { is_beta_chart } = client;
     const { is_dark_mode_on, is_mobile } = ui;
     const theme = is_dark_mode_on ? 'dark' : 'light';
     let yAxiswidth;
-    if (charts_ref?.chart) {
-        yAxiswidth = charts_ref.chart.yAxiswidth;
-    }
```

Comment:
we don't need yAxiswidth anymore? If not, we should also remove `let yAxiswidth;` line

Reply:
Done. Also removed string, where `yAxiswidth` was passed in props to the child component.